packages/fxa-settings/README.md

```diff
@@ -9,6 +9,75 @@ This documentation is up to date as of 2020-08-03.
 - `yarn test` to run unit tests
 - `yarn storybook` to open Storybook
 
+### GQL and REST API Calls
+
+FxA Settings communicates primarily with the FxA GraphQL API through use of [Apollo Client](https://www.apollographql.com/docs/react/) to indirectly interact with the `fxa-auth-server`. Requests that simply retrieve data are called queries and all other interaction requests are called mutations. See the [GQL documentation](https://graphql.org/learn/) to learn more. See (the documentation for `fxa-graphql-api`)[https://github.com/mozilla/fxa/tree/main/packages/fxa-graphql-api)] to connect to the playground, a place to view the API docs and schema, and to write and test queries and mutations before using them in a component.
+
+While most API calls can be performed with GQL, there are a few calls that must directly communicate with the auth server (for now) for security reasons. To make REST API calls to the auth server, use the `useAuth` and `useAwait` hooks.
+
+### Global Application Data
+
+This application uses [Apollo client cache](https://www.apollographql.com/docs/react/caching/cache-configuration/) to store app-global state, holding both data received from the GQL server and local global state. You can see the shape of this data in the schema found within the `GetInitialState` query, noting that a `@client` directive denotes local data.
```

Comment:
> local global state

I think I know what you mean, as in "state set from within and available across the entire app" (right?), just wanted to point out that this might be confusing.

Reply:
Yes, I'll change this to "local data" since that's what the GQL docs call it instead and link here: https://www.apollographql.com/docs/tutorial/local-state/